.github/workflows/run-tests.yml

```diff
@@ -22,13 +22,11 @@ jobs:
           python-version: '${{ matrix.python-version }}'
       - name: Install dependencies
         run: |
-          pip install codecov
           python scripts/ci/install
       - name: Run tests
         run: |
           python scripts/ci/run-tests --with-cov --with-xdist
       - name: Run codecov
-        run: |
-          rm tests/coverage.xml
-          mv tests/.coverage ./
-          codecov
+        uses: codecov/codecov-action@v2
```

Comment:
The current major version is v3. I think we should be using that instead here.

Reply:
Fixed in the most recent commit. Thanks!